frontend/src/metabase/components/select-list/SelectListItem.styled.jsx

```diff
@@ -30,7 +29,9 @@ const VERTICAL_PADDING_BY_SIZE = {
 };
 
 export const ItemRoot = styled.li`
-  display: flex;
+  display: grid;
```

Comment:
doing this so that I can use `justify-self: start` in `QuestionListItem` to move the `rightIcon` right next to the name of the question. `QuestionListItem` feels a little gross, but I couldn't think of a better alternative. I could pass a `className` of `justify-self-start` as a property of the `iconRight` object but since we are trying to move away from util classes that didn't feel right.